CRM/Dedupe/BAO/DedupeRuleGroup.php

```diff
@@ -128,6 +128,85 @@ public static function supportedFields($requestedType): array {
 
   }
 
+  /**
+   * Combine all the importable fields from the lower levels object.
+   *
+   * @deprecated - copy of importableFields to unravel.
+   *
+   * The ordering is important, since currently we do not have a weight
+   * scheme. Adding weight is super important
+   *
+   * @param int|string $contactType contact Type
+   *
+   * @return array
+   *   array of importable Fields
+   */
+  private static function importableFields($contactType): array {
+
+    if (TRUE) {
```

Comment:
@eileenmcnaughton since this function is being newly added I think it's worse for future whitespace cleanup to leave in this fake condition.

Reply:
@colemanw yeah I've removed the fake if - although now I see what the result of the function - I am thinking about how to generate that with apiv4 metadata - or just hard-code the damn thing! It does add in custom fields too.

Still probably worth merging this in case the rest doesn't happen

How useful is it to use

     'signature_html' => 'Signature Html',
          'signature_text' => 'Signature Text',